tests/func/test_import.py

```diff
@@ -73,9 +73,10 @@ def test_download_error_pulling_imported_stage(dvc_repo, erepo):
     dst_stage = Stage.load(dvc_repo, "foo_imported.dvc")
     dst_cache = dst_stage.outs[0].cache_path
 
-    shutil.rmtree(erepo.root_dir)
     os.remove(dst)
     os.remove(dst_cache)
 
-    with pytest.raises(DownloadError):
+    with patch(
+        "dvc.remote.RemoteLOCAL._download", side_effect=Exception
```

Comment:
Why the previous test didn't work for you?

Reply:
It relied on `CloneError`, which doesn't happen when you have cached external repo.